test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -41,7 +42,10 @@ public class ContentResultTest
             await contentResult.ExecuteResultAsync(actionContext);
 
             // Assert
-            Assert.Equal("text/plain; charset=utf-7", httpContext.Response.ContentType);
+            Assert.True(MediaTypeComparisons.AreEquivalent(
```

Comment:
This is not readable and `Assert.True` should be the last resort. Rather than adding all these `AreEquivalent()` overloads to the _product_, suggest a `MediaTypeAssertExtensions` class containing for example `AreEquivalent(this Assert assert, string, string, bool)` overloads. Best if that extensions class can live in a single project but use `Microsoft.AspNet.Mvc.TestCommon` if mandatory. Or do the `ExceptionAssert` / `TypeAssert` / `XmlAssert` thing and provide test helpers in a well-named `static` class.